backend/sirius-web-emf/src/test/java/org/eclipse/sirius/web/emf/services/validation/ValidatorTestUtils.java

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.emf.services.validation;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.eclipse.emf.common.util.BasicDiagnostic;
+import org.eclipse.emf.common.util.Diagnostic;
+
+/**
+ * Utility class to help testing validation.
+ *
+ * @author gcoutable
+ */
+public final class ValidatorTestUtils {
```

Comment:
Use a proper assertion class like we did it in other projects (see `DiagramAssertions`) instead of a Utils class. 99% of the time, if you are creating a `Utils` class, you are doing something wrong, it should have a proper name and goal instead of being the place where everything goes.